SUSE/rmt
# Final translation round

## Description

Last step in translations round. Enabling translations for the marked strings

#### How to test
* Set the language using the LANG variable while calling rmt-cli. Example: `LANG=de rmt-cli products ls`
* Ensure that the response is translated

## Change Type

*Please select the correct option.*


## Checklist

*Please check off each item if the requirement is met.*


## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.
